### PR TITLE
Add Numpad key for Kalimba Scale

### PR DESCRIPTION
**Pull request by Seongil-Yoon**

### CODE REVIEW

File: website/scripts/events-handler.js [hunk 1]
```diff
@@ -14,13 +14,14 @@ function parseKeyboardMappings(text)
...
-            return null;
-        }
+        console.log(lineSplit);
+        // if(lineSplit.length != 2)
+        // {
+        //     publichAlert("Bad formatting: "+line, 'danger')
+        //     publichAlert("Format should look like: key:note_id  ", 'warning')
+        //     publichAlert("Example: a:60  ", 'warning')
+        //     return null;
+        // }
```
Comment (jwdeveloper) [inline website/scripts/events-handler.js]: Is there any reason to remove alters?

File: website/index.html [hunk 2]
```diff
@@ -103,6 +103,7 @@
   if(localFileSource == false)
   {
     cssSource =  githubURL + cssSource
+    // cssSource =  "./" + cssSource
```
Comment (jwdeveloper) [inline website/index.html]: Please remove unused code

**Review by jwdeveloper — CHANGES_REQUESTED** [review]
Please resolve comments i left

File: website/scripts/events-handler.js (hunk 1 above)
Comment (Seongil-Yoon) [reply website/scripts/events-handler.js]: i have to  separate the number pad from the number.(Digit1 != Numpad1)
I got the code value, but the length exceeded 2, so I temporarily annotated it

File: website/index.html (hunk 2 above)
Comment (Seongil-Yoon) [reply website/index.html]: confirm

**Pull request merged**